org-code-javabuilder/playground/src/main/java/org/code/playground/TextItem.java

```diff
@@ -42,85 +53,99 @@ public TextItem(
    */
   public TextItem(String text, int x, int y, Color color, Font font, int height, double rotation) {
```

Comment:
you can have this constructor call the more specific constructor and set the default font style like this
```
this(text, x, y, color, font, FontStyle.NORMAL, height, rotation)
```
then you won't need to repeat code

Reply:
Nice, thanks!